### ISSUE TITLE
th: command not found

### ISSUE THREAD

**Issue by gitrobin** [junyanz/CycleGAN#82]
WHen I run DATA_ROOT=./datasets/horse2zebra name=horse2zebra_model th train.lua in the terminal,
it always remind me of  "th: command not found"

**Comment by denka0711** [comment]
@taesung89 why  Lua Torch, what's difference between Lua Torch and pytorch

**Comment by taesung89** [comment]
The deep learning framework, called Torch, was based on a programming language named Lua. This repository hosts the code for CycleGAN that was built on Torch. 

After the release of CycleGAN, people developed a new framework called PyTorch, which follows the style of Torch but runs using Python. Now that PyTorch became very popular, I called the original Torch Lua Torch to avoid confusion. 

So what I just meant was "You need to install Torch to run CycleGAN".

**Comment by denka0711** [comment]
hi @taesung89  thanks for your kind response, ! I am new one to Gan and pytorch. I am implementing your code, could you tell me which version just use the pytorch to implement your work (Object Transfiguration)

**Comment by tianke0711** [comment]
Hi me to

**Comment by tianke0711** [comment]
hi @taesung89 https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix is this version just use the pytorch?

**Comment by taesung89** [comment]
Yes, that link is correct. You should be able to run it using the current pytorch version. The installation instruction should be on that page. If you have problem running it, please look at the instruction and also issue on that git repo. Thanks!!

**Comment by tianke0711** [comment]
@taesung89  thanks

**Comment by AlexRMU** [comment]
###  Hard but easy
**Clone the Torch repo (forked to work with CUDA 10).**

`!git clone https://github.com/nagadomi/distro.git torch --recursive`

 **Install Torch.**
```
import os
os.chdir('./torch/')
!bash install-deps
!./install.sh
```

**Activate Torch.**

`!. ./install/bin/torch-activate`

**Make sure the installation worked.**

`!./install/bin/th`

Result:
```
  ______             __   |  Torch7 
 /_  __/__  ________/ /   |  Scientific computing for Lua. 
  / / / _ \/ __/ __/ _ \  |  Type ? for help 
 /_/  \___/_/  \__/_//_/  |  https://github.com/torch 
                          |  http://torch.ch 
	
th> ^C
```
